policy_tests/Makefile

```diff
@@ -47,6 +47,19 @@ bare-qemu_TIMEOUT = 360
 bare-qemu: CONFIG=bare-qemu
 bare-qemu: all
 
+# config for 64-bit bare-metal on QEMU
+bare64-qemu_SIM = qemu
+bare64-qemu_TESTS = bare64
+bare64-qemu_RUNTIME = bare64
```

Comment:
The runtime here should be `bare`, not `bare64`. The architecture should be an entirely independent option.

Reply:
Ok.  I think this might fix some of the duplicate entity/policy files as well.